entrypoint.sh

```diff
@@ -17,5 +17,10 @@ balena whoami > /dev/null
 # Test git is working
 git --version
 
+if [[ "${REGISTRY_SECRETS}" != "" ]]; then
+  mkdir -p "$HOME/.balena/"
+  echo ${REGISTRY_SECRETS} > "$HOME/.balena/secrets.json"
```

Comment:
It might be useful to pass the output of `echo ${REGISTRY_SECRETS}` through `jq .` so users can get immediate validation on any json syntax errors. Otherwise either the CLI or docker will fail with some cryptic error.